site/docs/patterns/form_multi_page.mdx

```diff
@@ -0,0 +1,123 @@
+
+### When should you use multi-page forms?
+- When the form can be divided into distinct themes. For example, "_Contact information_" -> "_Application_" -> "_Payment_" -> "_Review and send_".
+- When the form is very long.
+  - What is considered a high amount of form elements depends on the context. Usually the amount can be considered high if the user is required to scroll the view to see all form elements on a desktop sized screen.
+- When your users tend to do a lot of errors when filling out the form.
+
+### When should you not use multi-page forms?
+- When the amount of form elements is low. Dividing a small amount of inputs to multiple steps increases interaction cost of completing the form.
+  - What is considered a low amount of form elements depends on the context. Usually the amount of considered low if you can fit all fields on a single view without a need to scroll on a desktop sized screen.
+- When the user needs to compare or move information between multiple form steps.
+- When the form steps have a lot of relations. For example, a choice in a later form step must not make an earlier step erroneous.
+- When you do not know in which order the user wants to fill out the form. Multi-step forms are generally designed to be filled out in a specific order.
+  - Note! HDS multi-page forms support filling out the steps in a random order. However, this takes away many benefits that multi-page forms are designed to achieve.
+
+## Elements of a multi-page form
+HDS multi-page forms consist of three (3) elements:
+1. Form headings
+2. Form navigation
+3. Form area
+
+### 1. Form headings
+HDS multi-page forms feature two separate headings; the main heading and the step heading. 
+
+- The **form main heading** describes the whole form and it is located above the form navigation element. 
+- The **form step heading** describes the currently active step and it is located below the form navigation element.
+
+<Image src="../../static/patterns/form-multi-page/multi-page-form-headings@2x.png" alt="Form headings example" style="max-width:768px;" viewable />
+
+<div class="guideline-do" style="background: var(--color-success-light); padding: var(--spacing-s); margin-bottom: var(--spacing-layout-2-xs); max-width:600px;">
+    <div class="guideline-do-label" style="color:var(--color-success); margin-bottom:var(--spacing-s);"><IconCheck style={{ marginRight: 'var(--spacing-3-xs)', verticalAlign: 'middle'}} size="s"/><strong>Do</strong></div>
+    <ul style="margin-bottom:0;">
+        <li><strong>The step heading <em>must</em> include the number of the current step and the total amount of steps.</strong></li>
+        <li>When the form page changes, <strong>move the browser focus to the step heading.</strong></li>
+        <li>Customise heading levels if needed so that they are used semantically (<strong>h2</strong> after <strong>h1</strong> and so on)</li>
```

Comment:
Muuttaisin tämän niin päin että h1 on ennen h2sta, esim. "h1 is before h2", koska nyt tuo voi olla kognitiivisesti työläs kun elementti, jonka piti olla jälkeen, onkin ensin. Vähän kuin jos kirjoittaa sanan "vihreä" punaisella tekstillä, se saa aivot miettimään liikaa.